src/models/MeasurementPeriodValidator.ts

```diff
@@ -34,12 +34,10 @@ export const MeasurementPeriodValidator = Yup.object().shape({
     .when("measurementPeriodStart", (measurementPeriodStart, schema) => {
       if (measurementPeriodStart !== null) {
         if (!isNaN(measurementPeriodStart.getTime())) {
-          const dayAfter = new Date(
-            measurementPeriodStart.getTime() + 86400000
-          );
+          const dayAfter = new Date(measurementPeriodStart.getTime());
```

Comment:
I think this is not `dayAfter` anymore. Could be `day` or just place this inline in `schema.min`?